src/timeMachine/components/fluxFunctionsToolbar/FluxFunctionsToolbar.tsx

```diff
@@ -56,36 +52,22 @@ class FluxFunctionsToolbar extends PureComponent<Props, State> {
                     key={category}
                     category={category}
                     funcs={funcs}
-                    onClickFunction={this.handleClickFunction}
+                    onClickFunction={handleClickFunction}
                   />
                 ))
               }
             </TransformToolbarFunctions>
           </div>
         </DapperScrollbars>
-      </>
+      </ErrorBoundary>
     )
-  }
-
-  private handleSearch = (searchTerm: string): void => {
-    this.setState({searchTerm})
-  }
-
-  private handleClickFunction = (func: FluxToolbarFunction) => {
-    this.props.onInsertFluxFunction(func)
-  }
-}
-
-const mstp = (state: AppState) => {
-  const activeQueryText = getActiveQuery(state).text
-
-  return {activeQueryText}
```

Comment:
I didn't see any reason for this component to need the `activeQueryText` so I removed that dependency

Reply:
ok I lied, it's needed

Reply:
lied again, it's not needed. The issue was that we weren't re-rendering when props changed